### DISCUSSION TITLE
[Support]: Frigate log timestamp not in order or wrong 24h registration

### DISCUSSION THREAD

**Discussion by Floriszz** [blakeblackshear/frigate#15477] [General Support] [answered]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

The log timestamp goes from 2024-12-12 22:22:20h to 2024-12-12 10:50:37. See screenshot below.
![frigate logging timestamp off](https://github.com/user-attachments/assets/d985b7dd-1242-4075-8be1-08b438c2139f)

### Steps to reproduce

1. Open settings - System logs
2.  Click on Frigate tab
3. Scroll down or up where logs shift from 22h till 23h
...

### Version

0.14.1-f4f3cfa

### In which browser(s) are you experiencing the issue with?

Chrome 131.0.6778.109 (Official Build) (64-bit) and Microsoft Edge Version 131.0.2903.86 (Official build) (64-bit)

### Frigate config file

```yaml
Part of the config file that could be of value for this issue.

ui:
  # Optional: Set the time format used.
  # Options are browser, 12hour, or 24hour (default: shown below)
  time_format: 24hour
  # Optional: Set the date style for a specified length.
  # Options are: full, long, medium, short
  # Examples:
  #    short: 2/11/23
  #    medium: Feb 11, 2023
  #    full: Saturday, February 11, 2023
  date_style: medium
```

### docker-compose file or Docker CLI command

```yaml
docker run -d \
--name frigate \
--restart=unless-stopped \
--mount type=tmpfs,target=/tmp/cache,tmpfs-size=1000000000 \
--device /dev/dri/renderD128 \
--device /dev/bus/usb:/dev/bus/usb \
--shm-size=128m \
--privileged \
-v /volume2/docker/frigate/media:/media/frigate \
-v /volume2/docker/frigate/config:/config \
-v /etc/localtime:/etc/localtime:ro \
-v /dev/bus/usb:/dev/bus/usb \
-e LIBVA_DRIVER_NAME=i965 \
-e FRIGATE_RTSP_PASSWORD='yourpassword' \
-p 5055:5000 \
-p 8554:8554 \
-p 8555:8555/tcp \
-p 8555:8555/udp \
ghcr.io/blakeblackshear/frigate:stable
```

### Relevant Frigate log output

```shell
info
2024-12-11 21:42:04
frigate.app
Capture process started for Back: 525
info
2024-12-11 21:42:05
frigate.detectors.plugins.edgetpu_tfl
TPU found
error
2024-12-11 21:43:09
frigate.util.services
Unable to poll intel GPU stats: Failed to initialize PMU! (Invalid argument)
unknown
2024-12-12 22:31:47
unknown
24-12-11 21:43:09.606877014
info
2024-12-12 10:50:37
frigate.watchdog
Detection appears to be stuck. Restarting detection process...
info
2024-12-12 10:50:37
root
Waiting for detection process to exit gracefully...
info
2024-12-12 10:51:07
root
Detection process didn't exit. Force killing...
info
2024-12-12 10:51:07
root
Detection process has exited...
info
2024-12-12 10:51:07
detector.coral
Starting detection process: 6286
info
2024-12-12 10:51:07
frigate.detectors.plugins.edgetpu_tfl
Attempting to load TPU as usb
info
2024-12-12 10:51:10
frigate.detectors.plugins.edgetpu_tfl
TPU found
```

### Relevant go2rtc log output

```shell
info
2024-12-11 21:41:39
startup
Preparing new go2rtc config...
info
2024-12-11 21:41:40
startup
Not injecting WebRTC candidates into go2rtc config as it has been set manually
info
2024-12-11 21:41:40
startup
Starting go2rtc...
info
2024-12-11 21:41:40
startup
go2rtc platform=linux/amd64 revision=b2399f3 version=1.9.2
info
2024-12-11 21:41:40
startup
config path=/dev/shm/go2rtc.yaml
info
2024-12-11 21:41:40
rtsp
listen addr=:8554
info
2024-12-11 21:41:40
api
listen addr=:1984
info
2024-12-11 21:41:40
webrtc
listen addr=:8555
info
2024-12-11 21:41:49
startup
Starting go2rtc healthcheck service...
```

### Operating system

Other

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

Dahua, hikvision

### Screenshots of the Frigate UI's System metrics pages

![frigate system metrics](https://github.com/user-attachments/assets/43522a4b-8885-4af2-a858-10d232f954c0)

**Comment by NickM-27** [comment, marked as answer]
we can see in the screenshot that the log is actually from a different time (21:43:09). It is not clear what logged this, but this is not Frigate making this log and the sorting is done based on the time that the log was made not the timestamp that is shown on the log (which comes from S6)

**Reply by Floriszz** [reply]
ok clear. Thank you. Have to look further where this is coming from.
